src/state/pools/index.ts

```diff
@@ -59,6 +61,56 @@ const initialState: PoolsState = {
 }
 
 // Thunks
+const poolZero = poolsConfig.find((pool) => pool.sousId === 0)
```

Comment:
Suggestion - let's rename "pool zero" to "cake pool" or something. Otherwise it might be confused with IFO pool 0 (basic sale). At least I for a second thought about two IFO pools. 🤔

Reply:
I thought pool zero name is a more catchy for me. but cake pool is better for context 👍